index.d.ts

```diff
@@ -44,6 +44,21 @@ export type Instance = {
 
 export type Unmount = () => void;
 
+/**
+ * Hook that calls inputHandler callback with input that program received.
+ * Additionally contains helpful metadata for detecting when arrow keys were pressed.
+ */
+export function useInput(
+	inputHandler: (input: string, meta: Meta) => void
```

Comment:
```suggestion
	inputHandler: (input: string, metaKey: MetaKey) => void
```

Reply:
`meta` can be confused as `meta` keys, which is something different.

Reply:
Or maybe just call it `key` as that's what it really is. Not worth optimizing for a potential future where you add other things there.